lib/helper.js

```diff
@@ -16,8 +18,26 @@ function isEmpty(value) {
   return typeof value == 'string' && !value.trim() || typeof value == 'undefined' || value === null;
 }
 
+function encodeKey(metricName, tags) {
+  if (isEmpty(tags)) {
+    return metricName;
+  }
+  // Sort the tags based on their keys to ensure we don't duplicate metrics if the same tags come out of order.
+  var tagsJson = JSON.stringify(tags, Object.keys(tags).sort());
+  return metricName + tagSeparator + tagsJson ;
+}
+
+function decodeKey(key) {
+  if (key.indexOf(tagSeparator) > -1) {
+    return key.split(tagSeparator);
```

Comment:
From my understanding, this is very fragile.  Suppose I have a tag whose value is "-tags=".  What happens then?

Reply:
We currently use "-tags=" as the separator in our python sdk. So just wanted to be consistent.  the case of having an actual point tag name/value as "-tags=" seem rare but i would be happy to swap it with a better separator. can disuss this offline and update.